service/src/main/java/uk/gov/hmcts/reform/fpl/events/OrderEvent.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.fpl.events;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+
+@EqualsAndHashCode
+@Getter
+public class OrderEvent {
```

Comment:
still being used?

Reply:
seems that it is only used in tests and not main code

Reply:
removed